packages/app/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "app",
+  "version": "0.0.0",
+  "repository": "https://github.com/ImpeccableHQ/stackly-ui",
+  "license": "MIT",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "export": "next export",
+    "typecheck": "tsc --noEmit",
+    "clean": "rm -rf .vercel && rm -rf .turbo && rm -rf out && rm -rf .next"
+  },
+  "dependencies": {
+    "@stackly/ui": "*",
+    "@stackly/utils": "*",
+    "next": "canary",
```

Comment:
I think we should specify a version, like the latest 13.4 

https://nextjs.org/blog/next-13-4

Reply:
Added proper version